js/index.ts

```diff
@@ -0,0 +1,241 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ */
+
+import {useState, useEffect} from 'react';
+import DeprecatedUtils from './internal/deprecatedUtils';
+import DeprecatedSubscriptions from './internal/deprecatedSubscriptions';
+import * as DeprecatedTypes from './internal/deprecatedTypes';
+import Subscriptions from './internal/subscriptions';
+import * as Types from './internal/types';
+import NativeInterface from './internal/nativeInterface';
+
+const DEPRECATED_CHANGE_EVENT_NAME = 'connectionChange';
```

Comment:
this exists already in DeprecatedTypes - duplicate here?

Reply:
Unfortunately, you can't use a string constant type where you would use a variable in Typescript so it needs to be separate and duplicated. I have moved them to be next to each other in `DeprecatedTypes` though as it's a bit cleaner and clearer.